src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Consider adding a fallback font family for the `.stars__star` class to enhance cross-browser compatibility and ensure that text is rendered with a default font if the primary font fails to load. For example, `font-family: Arial, sans-serif;`.